package.json

```diff
@@ -152,11 +152,10 @@
     "sass": "^1.53.0",
     "sass-loader": "10.2.1",
     "storybook": "^7.1.0-alpha.1",
-    "stylelint": "15.10.1",
-    "stylelint-config-recommended": "^7.0.0",
-    "stylelint-formatter-pretty": "^3.1.0",
-    "stylelint-order": "^5.0.0",
-    "stylelint-scss": "^4.2.0",
+    "stylelint": "16.3.1",
+    "stylelint-config-standard-scss": "^13.0.0",
+    "stylelint-formatter-pretty": "^4.0.0",
+    "stylelint-order": "^6.0.0",
```

Comment:
I question the need for this package, but kept it since we wanted it at one point so it's gotta have a purpose. I just don't know how valuable it is specifying the order in CSS

Reply:
Honestly I don't love this lint rule. While it's kind of nice to see them in alphabetical order to find properties, it's actually kind of arbitrary, and there are good reasons to go against this in many cases.

Reply:
I can remove it if we're in agreement. 

Current config has order doing the following:

- alphabetizing rules
- ordering styles by custom properties > declarations > rules 
  - custom properties are CSS vars
  - declarations are like `display: block`
  - rules are nested rules (`a { span {} }`)
  - example of how this impacts css:
```css 
.button {
  // custom props
  --bg: red;
  --c: white;
  
  // declarations
  background-color: var(--bg);
  color: var(--c);

  // new rule
  svg {
    [declarations here]
  }
}
```

I personally don't see a benefit to alphabetizing and already write my CSS the way the order's specified in the 2nd bullet point.